pegasus/sites.v3/code.org/views/congrats_share.haml

```diff
@@ -1,9 +1,31 @@
+:javascript
+  // Triggers the registration of a Google Analytics "share_hoc_certificate" event.
+  // Args:
+  //   social_platform: The social platform the HOC certificate is being shared on. 
+  function trackSocialEvent(social_platform) {
+    // Determine the tutorial being shared via the document referrer.
+    if (!document.referrer) {
+      tutorial = "none";
+    } else if (document.referrer.indexOf("hoc") != -1) {
+      tutorial = "angrybirds";
+    } else if (document.referrer.indexOf("flappy") != -1) {
+      tutorial = "flappy";
+    } else if (document.referrer.indexOf("frozen") != -1) {
+      tutorial = "frozen";
+    } else if (document.referrer.indexOf("starwars") != -1) {
+      tutorial = "starwars";
+    } else {
+      tutorial = "unknown";
+    }
+    trackEvent(social_platform, "share_hoc_certificate", tutorial);
+  }
+
 #congrats-share-buttons
   %h2{:style=>'margin-top: 0; margin-bottom: 0;'}=I18n.t :share_achievement
 
   .share-buttons.clear
     %div#share-fb{:style=>"display:inline-block; margin-bottom: 10px; margin-right: 20px"}
-      %a.window-popup{:href=>"https://www.facebook.com/sharer/sharer.php?#{facebook.to_query}"}<
+      %a.window-popup{:href=>"https://www.facebook.com/sharer/sharer.php?#{facebook.to_query}", onclick: "trackSocialEvent('facebook'); return false;"}<
```

Comment:
Return false here will cause the browser not to navigate to the link, so I think the sharing won't actually happen.